application/controllers/Cron.php

```diff
@@ -88,7 +88,8 @@ public function run() {
 						echo "CRON: " . $cron->id . " -> is due: " . $isdue_result . "\n";
 						echo "CRON: " . $cron->id . " -> RUNNING...\n";
 
-						$url = base_url() . $cron->function;
+						$url = local_url() . $cron->function;
+						echo "CRON: " . $cron->id . " -> URL: " . $url . "\n";
```

Comment:
Since this is not necessary for the normal user I would depend this on the developement mode


```suggestion
                        if (ENVIRONMENT == "development") {
						    echo "CRON: " . $cron->id . " -> URL: " . $url . "\n";
                        }
```